cclib/bridge/cclib2openbabel.py

```diff
@@ -38,6 +38,7 @@ def makecclib(mol):
         attributes['atomcoords'].append([atom.GetX(), atom.GetY(), atom.GetZ()])
         attributes['atommasses'].append(atom.GetAtomicMass())
         attributes['atomnos'].append(atom.GetAtomicNum())
+        attributes['atomcoords'] = [attributes['atomcoords']]
```

Comment:
This should actually be indented to the same level as the return statement, otherwise you'll create an extra level of nesting every time an atom is read.